tests/test_models/test_subscriptions.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+Tests for the Subscriptions model.
+"""
+
+from forum.models import Subscriptions
+
+
+def test_get() -> None:
+    """Test get subscription from mongodb"""
+    subscriber_id = "test_subscriber_id"
+    source_id = "test_source_id"
+    source_type = "test_source_type"
+    Subscriptions().insert(
+        subscriber_id,
+        source_id,
+        source_type,
+    )
+    subscription_data = Subscriptions().get_subscription(subscriber_id, source_id)
```

Comment:
same as above, we can use only `subscription_id` to get subscription data if it exists. Also in tests below.

Reply:
wanted to to use the same methods as api for consistency and validation.